Minio/V4Authenticator.cs

```diff
@@ -514,6 +513,10 @@ private void SetContentMd5(IRestRequest request)
                 {
                     return;
                 }
+                // For insecure, authenticated requests calculate sha256 only
```

Comment:
Commenting just about sha256 in the md5 header function is out of place. It should say something like "MD5 is not required if in insecure and authenticated mode, since the sha256 header would be computed".